README.md

```diff
@@ -61,7 +62,8 @@ let client = Crux::new("127.0.0.1","3000").http_client();
 
 ## Http Client
 Once you have called `http_client` you will have an instance of the `HttpClient` struct which has a bunch of functions to query Crux on Docker and Standalone HTTP Server:
-* [`state`](https://docs.rs/transistor/1.3.2/transistor/http/struct.HttpClient.html#method.state) queries endpoint [`/`](https://opencrux.com/docs#rest-home) with a `GET`. No args. Returns various details about the state of the database.
+
+* [`state`](https://docs.rs/transistor/1.3.3/transistor/http/struct.HttpClient.html#method.state) queries endpoint [`/`](https://opencrux.com/reference/http.html#home) with a `GET`. No args. Returns various details about the state of the database.
```

Comment:
(https://docs.rs/transistor/1.3.3/transistor/http/struct.HttpClient.html#method.state) The resource doesn't exist

Reply:
version not released